chain/epoch-manager/src/adapter.rs

```diff
@@ -50,7 +50,10 @@ pub trait EpochManagerAdapter: Send + Sync {
     fn get_next_epoch_id_from_prev_block(&self, parent_hash: &CryptoHash)
         -> Result<EpochId, Error>;
 
-    /// Get epoch start for given block hash.
+    /// Get [`EpochId`] from a block belonging to the epoch.
+    fn get_epoch_id(&self, hash: &CryptoHash) -> Result<EpochId, Error>;
```

Comment:
should we call the argument 'block_hash' rather than hash ?

Reply:
Good catch! I specifically wanted to make this consistent, but fixed only one of the three places!